go/libkb/globals.go

```diff
@@ -127,6 +127,13 @@ type GlobalContext struct {
 	ActiveDevice *ActiveDevice
 
 	NetContext context.Context
+
+	// Initialization variables for idempotency
+	isInitDone               bool
```

Comment:
If this is only done once and never reset back to `false`, can we use `sync.Once` here instead?

Reply:
I guess so, but some of them can error out, in which case it shouldn't get set. So some will be booleans and some will be `sync.Once`.

Reply:
I see. For booleans, do we need worry about locking?

Reply:
Yeah, we probably do.

Reply:
I'll make it all `sync.Once`, and then for the error ones, I'll use a bool + mutex.

Reply:
Sounds good; thanks!